src/Compute/Compute.Test/ScenarioTests/VirtualMachineTests.ps1

```diff
@@ -2994,6 +2995,66 @@ function Test-VirtualMachineGetStatus
     }
 }
 
+<#
+.SYNOPSIS
+Test Virtual Machines
+#>
+function Test-VirtualMachineGetStatusWithHealhtExtension
+{
+    # Setup
+    $rgname = Get-ComputeTestResourceName
+
+    try
+    {
+        # Common
+        if ($loc -eq $null)
```

Comment:
$loc will always be null here. Just use the assignment line below this instead of having an if check.

Reply:
True! Resolving.